### ISSUE TITLE
Addig a new group in tiling mode 

### ISSUE THREAD

**Issue by str** [projectdelphai/panorama-tab-groups#175]
I tried this new feature. When view is in tiling mode, and I press the plus button to add a new group, the view mode changes to free-mode again.

**Comment by projectdelphai** [comment]
Does clicking the tiling button retile it? Right now, hitting the plus button just adds an immovable group in the left hand column and then hitting the tiling button retiles that. Just want to make sure we're seeing the same problem.

Also please update to 0.8.11 in case you haven't since there was a commit that broke a lot of things.

**Comment by str** [comment]
> Does clicking the tiling button retile it?
Yes, it changes to tiling and pushes the new group to the right/bottom corner.

> hitting the plus button just adds an immovable group in the left hand column
No. Hitting the plus button, sets the view mode as "free mode", and then adds a group in the left upper corner. I as expecting the tiling mode to stay, and add a new group. 

> Also please update to 0.8.11
Yes, tested in 0.8.11

**Comment by Felicis** [comment]
Just installed the addon (it's fantastic, thanks a lot @projectdelphai) and I experience the same behaviour as @str:
When adding a new group in tiling mode, the new group is added free-floating in the top left . Hitting the tiling button once corrects this. I think this could be fixed by automatically re-tiling the groups after hitting the new group button (if in tiling mode). If you are busy I can try to fix it myself and submit a pull request.

Tiled groups before hitting "new group" button:
![Selection_496](https://user-images.githubusercontent.com/11877488/121228947-aa6ac500-c88d-11eb-98e4-3e1a7f2e5f41.png)

Floating new group after hitting "new group" button (btw: since tiling is still active, the floating group cannot be moved around or resized):
![image](https://user-images.githubusercontent.com/11877488/121229068-cec6a180-c88d-11eb-86ce-af49eadb798a.png)
